### PR TITLE
API-11635 F: Vet Center URL Updates

### PR DESCRIPTION
**Pull request by giangnguyen2**

### CODE REVIEW

File: facilities/src/main/resources/websites.csv [hunk 1]
```diff
@@ -278,6 +293,8 @@ vc_0612V,https://www.va.gov/oakland-vet-center/
 vc_0613V,https://www.va.gov/high-desert-vet-center/
 vc_0614V,https://www.va.gov/chula-vista-vet-center/
 vc_0615V,https://www.va.gov/san-jose-vet-center/
+vc_0616v,https://www.va.gov/american-samoa-vet-center
```
Comment (as-va) [inline facilities/src/main/resources/websites.csv:296]: Curious that this one has a lower case `v`. The actual ID for this site in the API has a capital so let's update this to be consistent.

Comment (as-va) [reply facilities/src/main/resources/websites.csv:296]: Also, this line, the following, and one more further down are missing a `/` at the end of the path. This makes no difference as far as reachability of the link, but we should add it to these three so they stay consistent with the formatting of the rest of the links in the API.

Comment (giangnguyen2) [reply facilities/src/main/resources/websites.csv]: Updated lower case v  to V and added  `/` to the end of URLs

Comment (as-va) [reply facilities/src/main/resources/websites.csv]: I see the slashes now, but the `v` is still lowercase. Is there a commit that didn't get pushed?

Comment (giangnguyen2) [reply facilities/src/main/resources/websites.csv]: My bad. I  updated the supplied url files and re-ran the program not realizing that the TreeMap put method use case-insensitive comparison so the `v` didn't get updated. It should be good now

**Review by as-va — APPROVED** [review]

**Pull request merged**